### DISCUSSION TITLE
Rivet extension not working

### DISCUSSION THREAD

**Discussion by deependujha** [paradigmxyz/rivet#71] [General]
I am trying to setup rivet locally. I've installed all the dependencies and done with all the required steps.

But, when I run the `bun run dev` command. Sometimes, everything works perfectly and I'm able to test the local blockchain. But, majority of the time, it fails. Only webpage is shown and seems like extension is not working.
![Screenshot from 2023-09-20 10-17-19](https://github.com/paradigmxyz/rivet/assets/76887609/2bbfe208-0a34-49c3-9f2e-559a628069b2)

These are the errors in the console:
![Screenshot from 2023-09-20 10-17-55](https://github.com/paradigmxyz/rivet/assets/76887609/9f8cf23b-dcfa-42da-8a55-63228e4acefb)

The main issue I have is that, it works for sometimes and doesn't for most time.

#### Some details:
OS - Ubuntu 22.04.3 LTS
bun version - 1.0.1
anvil version - anvil 0.2.0 (ae89c92 2023-09-15T00:16:50.563312603Z)
chrome version - Version 117.0.5938.88 (Official Build) (64-bit)

**Comment by jxom** [comment]
Have you pulled the latest `main` changes? We added a change yesterday so `bun run dev` opens up a fresh chrome profile with the extension already installed... 

The errors in that trace there seems to be happening because you have multiple dev servers running. Try kill those servers (`killall -9 node`) and try again?

**Reply by deependujha** [reply]
yes, I've all the latest changes, except the one that you just made 15 minutes ago. I also tried `killall -9 node`.

But, now I'm getting the error:
![Screenshot from 2023-09-20 10-30-34](https://github.com/paradigmxyz/rivet/assets/76887609/6a2273c3-96d4-47f9-a8d0-0ca535e4470e)
